libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -196,6 +196,14 @@ const AP_Param::GroupInfo AP_MotorsHeli_Dual::var_info[] = {
     // @Increment: 1
     AP_SUBGROUPINFO(_swashplate2, "SW2_", 21, AP_MotorsHeli_Dual, AP_MotorsHeli_Swash),
 
+    // @Param: DCP_TRIM
+    // @DisplayName: Differential Collective Pitch Trim
+    // @Description: Used to trim DCP axis.  DCP_TRIM has the same units as the axis commanded output.  A value of 1 is full control output.  Trim is limited to 20 percent of output in one direction.
```

Comment:
As brought up by @Naterater perhaps a year ago, we should avoid putting "Used to" or "This Controls", etc at the beginning of a parameter description because all parameters are used to do something or control something. We can just skip right to the point and say, "DCP Trim.." or something like that